### ISSUE TITLE
[neovim] anually adding parsers can be solved with this

### ISSUE THREAD

**Issue by AusCyberman** [mjlbach/nix-dotfiles#5]
I just discovered this to avoid manually adding all parsers to nix to manage tree-sitter parsers. 
````nix
postFixup = "patchelf --add-needed ${prev.stdenv.cc.cc.lib}/lib/libstdc++.so.6 $out/bin/nvim";
```

**Comment by mjlbach** [comment]
Thanks! this is pretty cool, what happens if libc updates with a compiled parser? I assume ABI mismatch?

**Comment by AusCyberman** [comment]
Yeah, but you can just recompile parsers. minimal hassle. however, i have actually run into a problem. the libstdc++ doesnt include GLIBCXX_3.4.29 at all. but the haskell and yaml parsers are compiled with it.

**Comment by AusCyberman** [comment]
this can be fixed by changing the compiler to `gcc_latest.cc.lib`
